GameLauncher/App/MainScreen.cs

```diff
@@ -1857,10 +1857,10 @@ private void settingsGameFiles_Click(object sender, EventArgs e)
                                 if (exitCode == -1073740972)    errorMsg = "Game Crash: Debugger crash (0x" + exitCode.ToString("X") + ")";
                                 if (exitCode == -1073741676)    errorMsg = "Game Crash: Division by Zero (0x" + exitCode.ToString("X") + ")";
 
-                                if (exitCode == 1)              errorMsg = "You just killed nfsw.exe via Task Manager";
+                                if (exitCode == 1)              errorMsg = "The process nfsw.exe was killed via Task Manager";
```

Comment:
Actually, this can also happen when ModManager.asi kills the application due to crash

Reply:
This should check if ModNetReloaded is used, otherwise show that actual errorMsg

Reply:
Yes ... but that's still technically "The Launcher" from a players perspective, not a "YOU killed it" with the 'but I didn't even have Task manager open!" reply.